com.hoptodesk.HopToDesk.json

```diff
@@ -0,0 +1,264 @@
+{
+  "id" : "com.hoptodesk.HopToDesk",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "23.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "command" : "hoptodesk",
+  "modules" : [
+    {
+      "name" : "libappindicator",
```

Comment:
Use it from shared modules https://github.com/flathub/shared-modules?tab=readme-ov-file#usage

Reply:
Ok, updated for using shared-modules